src/components/AppRoute.jsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import { Route, Redirect } from 'react-router'
 
-import App from './App'
+import Layout from './Layout'
 import FileExplorer from '../containers/FileExplorer'
-import Files from '../containers/Files'
-import Trash from '../containers/Trash'
+import { Container as Files } from '../ducks/files'
+import { Container as Trash } from '../ducks/trash'
 
 const AppRoute = (
-  <Route component={App}>
+  <Route component={Layout}>
```

Comment:
We have to be careful when refactoring the front codebase as the mobile's one depends on it, cf. https://github.com/cozy/cozy-files-v3/blob/master/mobile/src/components/MobileAppRoute.jsx#L15